docs/hub/models-download-stats.md

```diff
@@ -2,156 +2,47 @@
-            },
-        ],
-    },
-    "ml-agents": {
-        filter: [
-            {
-                wildcard: { path: "*.onnx" },
-            },
-        ],
-    },
-    "nemo": {
-        filter: [
-            {
-                wildcard: { path: "*.nemo" },
-            },
-        ],
-    },
-    "open_clip": {
-        filter: [
-            {
-                wildcard: { path: "*pytorch_model.bin" },
-            },
-        ],
-    },
-    "sample-factory": {
-        filter: [
-            {
-                term: { path: "cfg.json" },
-            },
-        ],
-    },
-    "paddlenlp": {
-        filter: [
-            {
-                term: { path: "model_config.json" },
-            },
-        ],
-    },
-    "speechbrain": {
-        filter: [
-            {
-                term: { path: "hyperparams.yaml" },
-            },
-        ],
-    },
-    "sklearn": {
-        filter: [
-            {
-                term: { path: "sklearn_model.joblib" },
-            },
-        ],
-    },
-    "spacy": {
-        filter: [
-            {
-                wildcard: { path: "*.whl" },
-            },
-        ],
-    },
-    "stanza": {
-        filter: [
-            {
-                term: { path: "models/default.zip" },
-            },
-        ],
-    },
-    "stable-baselines3": {
-        filter: [
-            {
-                wildcard: { path: "*.zip" },
-            },
-        ],
-    },
-    "timm": {
-        filter: [
-            {
-                terms: { path: ["pytorch_model.bin", "model.safetensors"] },
-            },
-        ],
-    },
-    "diffusers": {
-        /// Filter out nested safetensors and pickle weights to avoid double counting downloads from the diffusers lib
-        must_not: [
-            {
-                wildcard: { path: "*/*.safetensors" },
-            },
-            {
-                wildcard: { path: "*/*.bin" },
-            },
-        ],
-        /// Include documents that match at least one of the following rules
-        should: [
-            /// Downloaded from diffusers lib
-            {
-                term: { path: "model_index.json" },
-            },
-            /// Direct downloads (LoRa, Auto1111 and others)
-            {
-                wildcard: { path: "*.safetensors" },
-            },
-            {
-                wildcard: { path: "*.ckpt" },
-            },
-            {
-                wildcard: { path: "*.bin" },
-            },
-        ],
-        minimum_should_match: 1,
-    },
-    "peft": {
-        filter: [
-            {
-                term: { path: "adapter_config.json" },
-            },
-        ],
-    }
+## How are `GGUF` files handled?
+
+GGUF files are not tied to a single library, so all of them are counted for downloads. This will double count downloads in the case a user performs cloning of a whole repository, but most users and interfaces download a single GGUF file for a given repo.
```

Comment:
```suggestion
GGUF files are self-contained and are not tied to a single library, so all of them are counted for downloads. This will double count downloads in the case a user performs cloning of a whole repository, but most users and interfaces download a single GGUF file for a given repo.
```